addons/web/static/src/js/views/pivot/pivot_view.js

```diff
@@ -71,8 +71,9 @@ var PivotView = AbstractView.extend({
             // the measure should be allowed.  However, be careful if you define
             // a measure in your pivot view: non stored functional fields will
             // probably not work (their aggregate will always be 0).
-            if (field.type === 'measure' && !(field.name in measures)) {
-                measures[field.name] = field;
+
+            if (field.attrs.type === 'measure' && !(field.attrs.name in measures)) {
+                measures[name] = fields[name];
```

Comment:
i think you should use here field.attrs.name in both place: this is the 'raw' field name, and this is the one you want (look up in line 64: the name variable can be decorated with an interval

Reply:
yes, indeed. I will update the commit